livre/eq/render/RayCastRenderer.cpp

```diff
@@ -253,6 +233,50 @@ struct RayCastRenderer::Impl
             _computedSamplesPerRay = std::max( maxVoxelsAtLOD, (float)minSamplesPerRay );
         }
 
+        Vector2f dataSourceRange;
+        uint32_t shaderDataType;
+        switch( _dataSource.getVolumeInfo().dataType )
```

Comment:
this switch is too long to be inlined and should be moved back to one or two separate methods.